resources/views/components/flight-button.blade.php

```diff
@@ -0,0 +1,6 @@
+<button
+    id="add_flight_btn"
+    class="px-4 py-2 rounded text-sm font-medium bg-green-600 text-white hover:bg-green-700"
+>
+Add Flight
```

Comment:
pass the id and text as props to the component, so its a reusable button component